src/main/kotlin/com/example/mykku/image/S3ImageUploadService.kt

```
@@ -123,19 +118,13 @@ class S3ImageUploadService(
 
             Pair(bufferedImage.width, bufferedImage.height)
         } catch (e: ImageException) {
-            // 도메인 예외는 그대로 전달
+            
             throw e
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)